src/aeso_compiler.erl

```diff
@@ -137,8 +138,9 @@ from_string1(aevm, ContractString, Options) ->
     {ok, maybe_generate_aci(Res, FoldedTypedAst, Options)};
 from_string1(fate, ContractString, Options) ->
     #{ fcode := FCode
+     , fcode_env := #{child_con_env := ChildContracts}
      , folded_typed_ast := FoldedTypedAst } = string_to_code(ContractString, Options),
-    FateCode = aeso_fcode_to_fate:compile(FCode, Options),
+    FateCode = aeso_fcode_to_fate:compile(ChildContracts, FCode, Options),
```

Comment:
Here would be a good place to compute source hashes for child contracts and store them in the `ChildContracts` map.

Reply:
Let's move it to the next iteration